.eslintrc.json

```diff
@@ -69,6 +69,7 @@
         "no-param-reassign": 1,
         "import/no-unresolved": 0,
         "no-shadow": 1,
+        "no-return-assign": 1,
```

Comment:
what is it ?

Reply:
According to eslint rules i cant use keyword 'return' in arrow function